### ISSUE TITLE
[Feature Request] MFR Safari Net Blacklist

### ISSUE THREAD

**Issue by circuitbomb** [powercrystals/MineFactoryReloaded#322]
An option in the configuration file to disable specified mob types from being captured by the Safari Net would assist in helping server administrators balance auto-spawner farms.

Ideally, entities could be listed by enityID:\* for all of the same type or entityID:typeID (e.g. for distinguishing normal Skeletons and Wither Skeletons) if multiple entities share the same ID, similar to how some mods or plugins disable blocks or their use.

**Comment by Kaltaros** [comment]
this is a great idea!

**Comment by AEnterprise** [comment]
yeah, i recently had a troll that used a safari net to capture an ender dragon and wither boss and relased them at the spawn

**Comment by Slind14** [comment]
+1
